pom.xml

```diff
@@ -382,14 +388,18 @@
 			<version>3.1</version>
 		</dependency>
 
-		<!-- for testing, uncomment to use junit tests in src/test/java -->
+		<!-- For testing -->
 		<dependency>
 			<groupId>javax.servlet</groupId>
-			<artifactId>servlet-api</artifactId>
-			<version>2.5</version>
+			<artifactId>javax.servlet-api</artifactId>
```

Comment:
I think this one can be removed, given that it is included in middleware. @abatac